clang/lib/Headers/openmp_wrappers/__clang_openmp_device_functions.h

```diff
@@ -15,9 +15,19 @@
 #endif
 
 #ifdef __cplusplus
+#include <cstddef>
```

Comment:
I'm unsure if there's ever a reason to include the C++ wrappers in a freestanding environment. Without C++, these go straight to the `clang` resource directly. With the C++ versions they go through the system which hopefully ends up at the resource directory intact.

Reply:
this is not freestanding